libcloudforensics/providers/kubernetes/base.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Kubernetes core class structure."""
+
+import abc
+from typing import List, TypeVar, Callable, Optional
+
+from kubernetes import client
+
+from libcloudforensics.providers.kubernetes import selector
+
+
+class K8sClient(metaclass=abc.ABCMeta):
+  """Abstract class representing objects that use the Kubernetes API."""
```

Comment:
`Attributes` section missing

Reply:
It seems that this is required when public attributes exist https://google.github.io/styleguide/pyguide.html#384-classes